magento/pwa-studio
# [Fix bug]: Filter and Sort buttons should display at same time

## Description

Display 2 buttons Sort & Filter at category page to the same time. The condition show Sort only filter data from query is available

## Related Issue





Closes #2677.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to any category and observe button display.
2. Filter & Sort button display at the same time now.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
